Modules/.gitignore

```diff
@@ -0,0 +1,9 @@
+Activity/
```

Comment:
Please don't commit this file.

Reply:
Okay, I will remove this and modified compose.json before committing finally because these are our internal companies module. I will adjust it.

Reply:
yeah no worries, I guess as much 😄 
For your ease, it might be easier to have a separate asgardcms installation only for the contributions.
That way you don't risk mixing both together by accident.